tests/test_dependencies.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+
+from docket import CurrentDocket, CurrentWorker, Docket, Worker
+from docket.dependencies import Retry
+
+
+async def test_dependencies_may_be_duplicated(docket: Docket, worker: Worker):
+    called = False
+
+    async def the_task(
+        a: str,
+        b: str,
+        docketA: Docket = CurrentDocket(),
+        docketB: Docket = CurrentDocket(),
+        workerA: Worker = CurrentWorker(),
+        workerB: Worker = CurrentWorker(),
```

Comment:
should these also be `single=True`? I guess theres no harm?

Reply:
Yeah I didn't want to constrain anything unnecessarily.  Having more than one `Retry` is actually conflicting info, but these are harmlessly redundant